### PR TITLE
Benchmarks script improvements

### PR DESCRIPTION
**Pull request by muharem**
The updated scripts are backward compatible. 

Updates:
- use pallet list command instead static list (easy to forget to update the list, e.g. pallet_xcm is missing now);
- additional arguments with default values (useful for local runs).

### CODE REVIEW

File: scripts/benchmarks-ci.sh [hunk 1]
```diff
@@ -1,70 +1,25 @@
...
-steps=50
-repeat=20
 category=$1
 runtimeName=$2
 artifactsDir=$3
+steps=${4:-50}
+repeat=${5:-20}
 
 benchmarkOutput=./parachains/runtimes/$category/$runtimeName/src/weights
 benchmarkRuntimeName="$runtimeName-dev"
```
Comment (bkontur) [inline scripts/benchmarks-ci.sh:10]: @muharem 
what about renaming `benchmarkRuntimeName` -> `benchmarkRuntimeChain` ?

**Review by EgorPopelyaev — APPROVED** [review]

**Comment by muharem** [comment]
bot rebase

File: scripts/benchmarks-ci.sh (hunk 1 above)
Comment (gilescope) [reply scripts/benchmarks-ci.sh:10]: runtimes have names. chains are run by nodes and run runtimes.

**Comment by muharem** [comment]
bot merge

**Review by gilescope — APPROVED** [review]

File: scripts/benchmarks-ci.sh [hunk 2]
```diff
@@ -1,70 +1,25 @@
...
-                pallet_xcm_benchmarks::fungible
-	)
+# Load all pallet names in an array.
+pallets=($(
+  ${artifactsDir}/polkadot-parachain benchmark pallet --list --chain="${benchmarkRuntimeName}" |\
+    tail -n+2 |\
+    cut -d',' -f1 |\
+    sort |\
+    uniq
+))
```
[the remark this answers is not in the document]
Comment (muharem) [reply scripts/benchmarks-ci.sh:19]: no, it does not. if the pallet not listed in define_benchmarks macro, it wont be returned by the command

**Pull request merged**